logger-interface/src/logger/ConsoleLogger.java

```diff
@@ -0,0 +1,12 @@
+package logger;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class ConsoleLogger {
+    public void getConsolLogger(String name) {
+        Logger logger = Logger.getLogger(ConsoleLogger.class.getName());
+        logger.log(Level.INFO, name);
+
```

Comment:
These loggers should implement a common interface, as shown in the UML. Reopening your task